docs/roadmap.md

```diff
@@ -509,10 +509,18 @@ def pick_func(cond):
         def bar0(): return 42
         result = bar0
     else:
-        def bar1(): return 3 # error: redeclaration of 'bar'
+        def bar1(): return 3 
         result = bar1
     return result
 ```
+In mojo 0.6.0, this yields:
+```
+pick_func.mojo:9:12: error: cannot implicitly convert 'fn() raises capturing -> object' value to 'object' in return value
```

Comment:
I think this is just an error in the code example. It appears that `pick_func()` needs to indicate that it returns a function:

```
def pick_func(cond) -> def() capturing:
```
Then the rest of the example should work fine.